examples/worlds/rf_comms.sdf

```diff
@@ -0,0 +1,190 @@
+<?xml version="1.0" ?>
+<!--
+  Ignition Gazebo rfcomms plugin demo.
+
+  Compile the comms publisher:
+    cd ign-gazebo/examples/standalone/comms
+    mkdir build
+    cd build
+    cmake ..
+    make
+
+  Try launching a comms subscriber:
+    ign topic -e -t addr1/rx
+
+  Try launching a comms publisher:
+    ./publisher addr1
```

Comment:
should this be `addr2` since the src address is hardcoded to `addr1` in `publisher.cc`?

Reply:
also mention try moving one of the boxes away to see msg drops. In this example, I start to see msg drops when I move `box1` to  `x=~300`

Reply:
Changed to `addr2` in [991b690](https://github.com/ignitionrobotics/ign-gazebo/pull/1428/commits/991b6906a169f4d63bbcdf5b4bd72ace57848d6b).

That's the expected behavior. Msgs should drop more often if we increase the bandwidth but that's about right for exterior. In any case this is all configurable. For example, the parameter `fading_exponent` has a direct impact in the usable range.